libenv/sysinfo.c

```diff
@@ -1536,6 +1536,9 @@ static void OSClasses(EvalContext *ctx)
 #if defined(__ANDROID__)
     SetFlavor(ctx, "android");
 #endif
+#if defined(__TERMUX__)
+    SetFlavor(ctx, "termux");
+#endif
```

Comment:
This would be nicer as
```
#if defined(__TERMUX__)
...
#elif defined(__ANDROID__)
...
#endif
```

Reply:
I need `__TERMUX__` to override `__ANDROID__` so it has to be an if/endif then if/endif.